waku/v2/waku_rln_relay/rln/wrappers.nim

```diff
@@ -53,14 +53,14 @@ proc membershipKeyGen*(ctxPtr: ptr RLN): RlnRelayResult[IdentityCredential] =
 
   return ok(identityCredential)
 
-type RlnTreeConfig* = ref object of RootObj
+type RlnTreeConfig = ref object of RootObj
   cache_capacity*: int
```

Comment:
@rymnc - Sorry to be picky but could these be "hidden" as well?

Reply:
Sure, addressed in 358bba19!